Add theme hugo-theme-relearn

McShelby opened the pull request.

McShelby commented:
Is anything wrong about my PR? Just wondering because all other open PRs but mine are merged?

Just for information: I forked the original Learn theme, because it's stalled for quite some while.

digitalcraftsman commented:
Hello Sören,

thanks for your theme submission. Relearn seems to be a fork of the [learn theme](https://github.com/matcornic/hugo-theme-learn) that is also listed on www.themes.gohugo.io. Hugo users could find both themes but will not immediately able to understand how they relate to each other, i.e. that relearn is a fork. It might also be of interest for them to know what changes or additions have been made in your fork. This kind of information should be visible in the README of your theme.

McShelby commented:
Thank for the information. I'll provide some additions in the README in the next few days.

McShelby commented:
> I'll provide some additions in the README in the next few days.

As always, software projects take longer than intended.

Because this theme is meant to be a drop-in replacement of the original learn theme, it visually looks very similar by intent.

I've added a short "motivation" statement in the [README.md](https://github.com/McShelby/hugo-theme-relearn#readme) and prominently in a notice box on the [front page of the theme's documentation](https://relearn.netlify.app/), giving the main reason behind my fork. This box even shows up in the screenshots, that will be shown in the theme repo.

Additionally I've added a lot of links to the original theme - even in the `theme.toml` giving credits to the orignial theme and providing a [changelog](https://relearn.netlify.app/basics/history/) and [migration instructions](https://relearn.netlify.app/basics/migration/).

Version 2.0.0 of my fork was released a few hours ago. It alone [fixes 27 issues](https://github.com/McShelby/hugo-theme-relearn/milestone/4?closed=1), many of them are long outstanding in the original learn theme. And that's only for the latest release - not counted the issues fixed in previous (smaller) releases.

Don't get me wrong: Forking the theme wasn't a light decision and loomed in me for almost a year. I've made [proposals on how to revive the development](matcornic/hugo-theme-learn#422) multiple times but with no success. Even other users are discussing the theme's state to no success (see matcornic/hugo-theme-learn#502, matcornic/hugo-theme-learn#505). I've made PRs, code reviews and giving help in the issues comments with almost no reaction from the mentainers.

Although the theme repo mentions the [last update](https://themes.gohugo.io/themes/hugo-theme-learn/) to be on 2021-03-31, the [commit history](https://github.com/matcornic/hugo-theme-learn/commits/master) shows a different picture.

I am more than willing to dump my fork if the development of the original learn theme comes up to speed again. [**But for the moment the learn theme is dead**](https://github.com/matcornic/hugo-theme-learn/issues/502#issuecomment-869602795).

I hope the information given, help to make a positive decision for including my fork in the theme repo.

McShelby commented:
@bep, @digitalcraftsman : Given my previous explanation, is there at least a chance to get listed in the theme gallery? Is there something else I can do to get listed?
